.github/workflows/ci.yml

```diff
@@ -45,6 +45,12 @@ jobs:
         curl -sLf https://github.com/bitnami/kubecfg/releases/download/v0.16.0/kubecfg-linux-amd64 >~/bin/kubecfg
         chmod +x ~/bin/kubecfg
 
+    - name: Set up Go 1.x
+      uses: actions/setup-go@v2
+      with:
+        go-version: ^1.17
+      id: go
```

Comment:
Why is this required? I don't thinks it's required for building the container.

Reply:
Yes. We added that as a result of a build failure.

It appears that without this GH used a go compiler older than 1.16.

We choose to pin 1.17 because it's better to build the docker image with the latest version as opposed to the minimal supported version.

Reply:
yeah, there's no reference to a default version in [actions/setup-go](https://github.com/actions/setup-go#v2) so i'm guessing it's defaulting to whatever's installed in ubuntu 20.04.3

Reply:
thanks for the context